google-site-kit.php

```diff
@@ -45,7 +46,15 @@ function googlesitekit_activate_plugin( $network_wide ) {
 	if ( version_compare( PHP_VERSION, GOOGLESITEKIT_PHP_MINIMUM, '<' ) ) {
 		wp_die(
 			/* translators: 1: version number */
-			esc_html( sprintf( __( 'Site Kit requires PHP version %s', 'google-site-kit' ), GOOGLESITEKIT_PHP_MINIMUM ) ),
+			esc_html( sprintf( __( 'Site Kit requires PHP version %s or higher', 'google-site-kit' ), GOOGLESITEKIT_PHP_MINIMUM ) ),
+			esc_html__( 'Error Activating', 'google-site-kit' )
+		);
+	}
+
+	if ( version_compare( get_bloginfo( 'version' ), GOOGLESITEKIT_WP_MINIMUM, '<' ) ) {
+		wp_die(
+			/* translators: 1: version number */
```

Comment:
The other comment needs to be fixed too, but there is already an issue for that 👍 
```suggestion
			/* translators: %s: version number */
```